cmd/gpu_plugin/fractional.md

```diff
@@ -1,6 +1,6 @@
-# GPU plugin with GPU Aware Scheduling
+# GPU plugin with GPU Aware Scheduling (deprecated)
 
-This is an experimental feature.
+This is a deprecated feature. It is recommended to instead use the newer [DRA resource driver](https://github.com/intel/intel-resource-drivers-for-kubernetes) to achieve fractional GPU resource allocating.
```

Comment:
This could use direct link to DRA GPU driver doc (I assume its location is now stable):
```suggestion
This is a deprecated feature. In Kubernetes v1.32+, [DRA GPU resource driver](https://github.com/intel/intel-resource-drivers-for-kubernetes/blob/main/doc/gpu/README.md) can be used for fractional GPU resource allocating instead.
```